src/content/en/2025/accessibility.md

```diff
@@ -3,18 +3,1190 @@
+
+<a hreflang="en" href="https://www.a11y-collective.com/blog/artificial-intelligence-accessibility/">The A11Y Collective blog on Artificial Intelligence and Accessibility</a> points out that while AI tools that automate `alt` text or real-time captions, and voice assistants can help accessibility at scale, they still struggle with accuracy, context, privacy, and bias.
+
+<a hreflang="en" href="https://dri.es/comparing-local-llms-for-alt-text-generation">Research by Dries Buytaert</a> shows AI can tackle huge backlogs of unlabeled images, but human review is still essential for quality. He explores the balance between quality, privacy, cost, and complexity for organizations considering AI-powered alt text.
+
+<a hreflang="en" href="https://www.digitalaccesstraining.com/pages/articles?p=ai-and-accessibility-opportunities-and-challenges-for-content-creators">Digital Accessibility Training</a> outlines the opportunities and challenges of AI for content creators. AI tools enable accessibility features at scale but raise concerns about content validity, bias, and ethical usage.
+
+Hidde de Vries <a hreflang="en" href="https://hidde.blog/ai-for-accessible-components/">contrasts how humans and language models approach accessible component code</a>. Humans base HTML, CSS, and ARIA decisions on specifications, user needs, assistive technology behavior, and platform quirks, all guided by intentions for the interface. LLMs instead predict likely code from training data, which is problematic because most existing code has accessibility issues, and the models lack intent or understanding of specific users.
+
+Adrian Roselli acknowledges that recent advances in computer vision and LLMs have brought real benefits, such as better image descriptions and improved captions and summaries. However, he argues <a hreflang="en" href="https://adrianroselli.com/2023/06/no-ai-will-not-fix-accessibility.html">these tools still lack context and authorship</a>. They can't know why content was created, what a joke or meme depends on, or how an interface is meant to work. Their descriptions and code suggestions can easily miss the point or mislead users.
+
+AI raises significant ethical concerns that go beyond accessibility.
+
+{{ figure_markup(
+  caption="Comparable number of gasoline cars for carbon emissions.",
+  content="100",
+  classes="big-number",
+)
+}}
+
+One major issue is environmental impact. A <a hreflang="en" href="https://arxiv.org/ftp/arxiv/papers/2104/2104.10350.pdf">study on carbon emissions and large neural network training</a> estimated that training a single large language model such as GPT‑3 consumed more than 1,200 megawatt-hours of electricity and produced around 500–550 metric tons of CO₂-equivalent. This is comparable to the lifetime emissions of over 100 gasoline cars.
```

Comment:
I'm not sure how this relates to a11y? But OK to include it. It would be nice to link to the Sustainability chapter though for those interested in the topic.

Reply:
I've linked to it, hopefully they will write about AI in the emerging technologies section. I wanted to make a note on their document as a comment, but can't :)